axum-extra/src/response/file_stream.rs

```diff
@@ -118,20 +118,24 @@ where
     /// Set the file name of the [`FileStream`].
     ///
     /// This adds the attachment `Content-Disposition` header with the given `file_name`.
+    #[must_use]
     pub fn file_name(mut self, file_name: impl Into<String>) -> Self {
         self.file_name = Some(file_name.into());
         self
     }
 
     /// Set the size of the file.
-    pub fn content_size(mut self, len: u64) -> Self {
+    #[must_use]
+    pub const fn content_size(mut self, len: u64) -> Self {
         self.content_size = Some(len);
         self
     }
 
     /// Return a range response.
     ///
+    /// ```rust
```

Comment:
I think you need to add `no_run` here so that it's not considered as a doctest.

But regardless, it's not valid rust either, so I'd probably mark it as such (maybe text instead of rust? Not sure).

Reply:
What the heck even is this line?